AndroidUI/resources/TestData.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If I remember correctly, this file already exists in UI project, so it's more clever to move this file in Core project

Reply:
Now I see that you attached expectedData.json to, so why this file exists?